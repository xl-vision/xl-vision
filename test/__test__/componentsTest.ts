describe('check component list', () => {
  it('list components in @xl-vision/react', () => {
    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment,global-require
    const components = require('../../packages/react/src');
    expect(Object.keys(components)).toMatchSnapshot();
  });
});
