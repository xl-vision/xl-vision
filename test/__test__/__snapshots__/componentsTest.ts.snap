// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check component list list components in @xl-vision/react 1`] = `
Array [
  "styled",
  "createGlobalStyles",
  "ThemeProvider",
  "Transition",
  "CSSTransition",
  "CollapseTransition",
  "TransitionGroup",
  "Row",
]
`;
